merkle/rfc6962/rfc6962.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rfc6962
```

Comment:
I agree. There's a lot of places in the code where this is either hard-coded, supposed to be looked up from the log config.  

Hopefully this change to putting this hashing algorithm in it's own package will make it easier to isolate the personality specific stuff later.